tests/contrib/codemeta/test_codemeta_custom_fields.py

```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+"""CodeMeta custom fields tests."""
+
+import pytest
+from invenio_access.permissions import system_identity
+
+from invenio_rdm_records.cli import (
+    create_records_custom_field,
+    custom_field_exists_in_records,
+)
+from invenio_rdm_records.contrib.codemeta.custom_fields import (
+    CODE_CUSTOM_FIELDS,
+    CODE_NAMESPACE,
+)
+from invenio_rdm_records.proxies import (
+    current_rdm_records_service as rdm_records_service,
+)
+from invenio_rdm_records.records import RDMDraft, RDMRecord
+
+
+def create_record(record_id):
+    """Uses rdm records service to publish a record."""
+    # Publish a record
+    return rdm_records_service.publish(system_identity, record_id)
+
+
+def create_draft(draft_data):
+    """Uses rdm records service to create a draft."""
+    return rdm_records_service.create(system_identity, draft_data)
+
+
+@pytest.fixture(scope="function")
+def codemeta_fields_record(minimal_codemeta_record):
+    """Creates a record containing CodeMeta fields."""
+    draft = create_draft(minimal_codemeta_record)
+    record = create_record(draft.id)
+    return record
+
+
+def test_custom_fields_loading(running_app, cli_runner, codemeta_cf_name):
```

Comment:
Should this be responsability of this module? The logic behind CF loading is in the cli (not contrib/codemeta), I think this module should assume it works (this is what triggered me to think if we could avoid using the CLI command in tests)

Reply:
My idea here was to test whether loading these specific custom fields do not yield any error. 
I do agree that it shouldn't be this module's responsability to test whether loading custom fields works or not, however I think that it's useful to have a test that validates the loading of CodeMeta CF.
In your opinion, would it be enough to simply create a draft and see if it fails? If so, I can remove this test.

Reply:
I'm convinced thanks! could you add this explanation in the test docstring or a comment? I see myself coming back to this again 😂

Reply:
I added a small comment in the test. I also ended up splitting this test into smaller ones.